.classpath

```diff
@@ -1,28 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah, that's right, but as it has already "lived" in the repo, so we can no "ignore" it with .gitignore, the proper step would be:

1) git rm --cached <filename>
2) commit it
3) add it to .gitignore